sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -642,84 +642,97 @@
                 {
                   "components": [
                     {
-                      "label": "Gender",
-                      "tableView": true,
-                      "persistent": "client-only",
-                      "calculateValue": "value = data.gender;",
-                      "key": "readOnlyGender",
-                      "attributes": {
-                        "data-cy": "gender"
-                      },
-                      "type": "textfield",
-                      "input": true,
-                      "hideOnChildrenHidden": false,
-                      "placeholder": "",
-                      "prefix": "",
-                      "customClass": "",
-                      "suffix": "",
-                      "multiple": false,
-                      "defaultValue": null,
-                      "protected": false,
-                      "unique": false,
-                      "hidden": false,
-                      "clearOnHide": true,
-                      "refreshOn": "",
-                      "redrawOn": "",
-                      "modalEdit": false,
-                      "dataGridLabel": false,
+                      "label": "Verified email",
                       "labelPosition": "top",
+                      "labelWidth": "",
+                      "labelMargin": "",
+                      "placeholder": "",
                       "description": "",
-                      "errorLabel": "",
                       "tooltip": "",
-                      "hideLabel": false,
+                      "prefix": "",
+                      "suffix": "",
+                      "widget": {
+                        "type": "input"
+                      },
+                      "displayMask": "",
+                      "customClass": "",
                       "tabindex": "",
-                      "disabled": false,
+                      "autocomplete": "",
+                      "hidden": false,
+                      "mask": false,
                       "autofocus": false,
+                      "spellcheck": true,
+                      "disabled": false,
+                      "tableView": true,
+                      "modalEdit": false,
+                      "multiple": false,
+                      "persistent": "client-only",
+                      "inputFormat": "plain",
+                      "protected": false,
                       "dbIndex": false,
+                      "case": "",
+                      "truncateMultipleSpaces": false,
+                      "encrypted": false,
+                      "redrawOn": "",
+                      "clearOnHide": true,
                       "customDefaultValue": "",
+                      "calculateValue": "value = data.email;",
```

Comment:
Not part of the PR, I noticed the `calculateValue` as it is here in other form.io definitions, unless I am missing something it is only needed if the `data` property is different than the input API name. If they are the same this `calculateValue` is useless.